### ISSUE TITLE
How can I get dialogue recordings with Monika?

### ISSUE THREAD

**Issue by gtitita** [Monika-After-Story/MonikaModDev#9483]
I need to get the recordings of the dialogue with my Monika for one project. But retyping from the game or from code files is very inconvenient. Therefore, if someone knows a way, then I would be very grateful if they explained it to me.

**Comment by Wingdinggaster656** [comment]
Not very sure what you mean.
You mean how to get all dialogues in MAS? Then try "_regular expression_".

**Comment by gtitita** [comment]
> regular expression

Yes, listen, I need Monica's dialogue, but you know, in the format of how they are recorded in the "story". And what is "regular expression"? I'm sorry, but I'm not very good at all this.

**Comment by Wingdinggaster656** [comment]
> > regular expression
> 
> Yes, listen, I need Monica's dialogue, but you know, in the format of how they are recorded in the "story". And what is "regular expression"? I'm sorry, but I'm not very good at all this.

https://en.m.wikipedia.org/wiki/Regular_expression

You could use regular expressions to extract Monika's dialogue from the source file.
